cherry/_torch.py

```diff
@@ -123,7 +123,7 @@ def onehot(x, dim):
     if isinstance(x, np.ndarray):
         size = x.shape[0]
         x = th.from_numpy(x).long()
-    if isinstance(x, (int, float)):
+    if (int, float, np.integer, np.float)):
```

Comment:
I think there's a small typo: this line should be `if isinstance(x, (int, float, np.integer, np.float)):`, and changing it should also fix the tests not passing.

Reply:
i am sorry. it was a typical copy & paste error. just fixed the typo and will add some tests.